mbf_abstract_nav/CMakeLists.txt

```diff
@@ -50,6 +50,12 @@ catkin_package(
   DEPENDS Boost
 )
 
+if("$ENV{ROS_DISTRO}" STREQUAL "melodic")
```

Comment:
We should make it the other way around. So for `kinetic` and `luna` the define `USE_OLD_TF`. New ros distros would be autometically included.

Reply:
sounds reasonable, yes.  always looking forward!

For the change on the action servers, remove the reference is the first thing I wanted to try, so... I feel confirmed.